### PR TITLE
VACMS-7250 - add MVP data change flags

### LINKED ISSUE #7250: Add MVP data change flags
## Description
As an concerned data watcher I need to have the following Flags available to be surfaced so that those like me can see what happened and apply define runbooks.

Should be done in Feature branch as this can not be rolled out by itself.

## UX design needs
- [ ] Placement of the flags in the node edit  (where in this and how)
image
- [ ] Flag names review.
- [ ] Input on these values for each flag
image

## Acceptance Criteria
- [ ] Flag "Changed title" (label and link text match)
  - Flaggable Types  
     - VAMC Facility
     - NCA Facility
     - VA Form
     - VBA Facility
     - Vet Center
     - Vet Center - Mobile Vet Center
     - Vet Center - Outstation
- [ ] Flag "Changed filename"  (label and link text match)
    - Flaggable Types   
     - VA Form
- [ ] Flag "Deleted"  (label and link text match)
    - Flaggable Types   
       - VA Form
- [ ] Flag "Removed from source"  (label and link text match)
  - Flaggable Types   
     - VAMC Facility
     - NCA Facility
     - VBA Facility
     - Vet Center
     - Vet Center - Mobile Vet Center
     - Vet Center - Outstation
 - [ ] Flag "New"  (label and link text match)
  - Flaggable Types   
     - VAMC Facility
     - NCA Facility
     - VBA Facility
     - Vet Center
     - Vet Center - Mobile Vet Center
     - Vet Center - Outstation
- [ ] Properties of each flag
  - [ ] Flag Link Text should match label
  - [ ]  Scope: Global
  - [ ] Display link as field: TRUE
  - [ ] Display Checkbox on entity edit form: TRUE
  - [ ] Display on Revision comparison view mode sounds like it would be a good thing to turn on, but I can see no evidence that this shows anywhere.
  - [ ] Link Type: Normal link (this is so it  will require a node save so revision log can show it.

###Display:
  - [ ] Not visible on node view
  - [ ] Visible in edit form   Residing in the workflow box near the log message and moderation state  
  
image

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

### PR DESCRIPTION
**Pull request by JayDarnell**
## Description

Closes #7250 

## Testing done

Visual testing

## Screenshots

New flags:
Screen Shot 2021-12-29 at 3 00 17 PM

VAMC Facility:
Screen Shot 2021-12-29 at 3 03 53 PM

NCA Facility:
Screen Shot 2021-12-29 at 3 04 41 PM

VA Form:
Screen Shot 2021-12-29 at 3 05 30 PM

VBA Facility: 
Screen Shot 2021-12-29 at 3 06 07 PM

Vet Center:
Screen Shot 2021-12-29 at 3 06 46 PM

Vet Center - Mobile Vet Center:
Screen Shot 2021-12-29 at 3 07 29 PM

Vet Center - Outstation:
Screen Shot 2021-12-29 at 3 11 30 PM

## QA steps

As an admin user
- [ ] Visit /admin/content
- [ ] Find a piece of content for each of the following types: (VAMC Facility, NCA Facility, VA Form, VBA Facility, Vet Center, Vet Center - Mobile Vet Center and Vet Center - Outstation)
- [ ] Edit the piece of content, scroll to the bottom and visually verify the existence of the flags within the editorial workflow section
- [ ] View the piece of content, verify that none of the flags are visible on node view

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### CODE REVIEW

File: config/sync/core.entity_form_display.node.va_form.default.yml [hunk 1]
```diff
@@ -281,6 +282,8 @@ content:
     settings:
       placeholder_url: ''
       placeholder_title: ''
+      linkit_profile: default
```
Comment (JayDarnell) [inline config/sync/core.entity_form_display.node.va_form.default.yml:285]: This was picked up automatically by configuration management but I'm not sure why.

File: config/sync/core.entity_view_display.node.nca_facility.teaser.yml [hunk 2]
```diff
@@ -16,14 +16,23 @@ targetEntityType: node
 bundle: nca_facility
 mode: teaser
 content:
-  links:
-    weight: 100
+  content_moderation_control:
```
Comment (JayDarnell) [inline config/sync/core.entity_view_display.node.nca_facility.teaser.yml:19]: I'm not sure why this wasn't previously captured. This matches configuration currently on production.

File: config/sync/core.entity_view_display.node.vba_facility.teaser.yml [hunk 3]
```diff
@@ -16,14 +16,23 @@ targetEntityType: node
 bundle: vba_facility
 mode: teaser
 content:
-  links:
-    weight: 100
+  content_moderation_control:
```
Comment (JayDarnell) [inline config/sync/core.entity_view_display.node.vba_facility.teaser.yml:19]: I'm not sure why this wasn't previously captured. This matches configuration currently on production.

**Comment by JayDarnell** [comment]
@kevwalsh @swirtSJW there are a few items in the config file changes above that didn't come from any changes I made but were picked up regardless. I've marked a few of them with comments above. Other changes I manually updated before creating this pull request (the revision log items that seem to be removed every time I run config export as one example).

**Review by swirtSJW — APPROVED** [review]
Jay this all looks great.  I am going to close the issue, but leave this in draft, so it becomes the integration branch.   This can't go live until we we work on the perms/display logic.

**Comment by swirtSJW** [comment]
Closing this as it has been merged into integration branch here https://github.com/department-of-veterans-affairs/va.gov-cms/pull/7424

**Pull request closed without merging**